src/QsCompiler/Transformations/ClassicallyControlled.cs

```diff
@@ -997,14 +1015,11 @@ public override QsStatementKind OnConditionalStatement(QsConditionalStatement st
                                 this.SharedState.IsConditionLiftable = false;
                             }
                         }
-
-                        this.SharedState.GeneratedOpParams = contextParams;
-                        this.SharedState.IsValidScope = contextValidScope;
                     }
 
                     if (this.SharedState.IsConditionLiftable)
                     {
-                        this.SharedState.GeneratedOperations?.AddRange(generatedOperations);
+                        this.SharedState.GeneratedCallables?.AddRange(generatedOperations);
```

Comment:
Is it valid for `this.SharedState.GeneratedCallables` to be null here?

Reply:
It should not be null here, as the GeneratedCallables should get a new List at the start of every namespace.

Reply:
Changed to use the null-forgiveness operator instead.